sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/GeneratedMutableEvaluationSuite.scala

```diff
@@ -44,14 +43,6 @@ class GeneratedMutableEvaluationSuite extends ExpressionEvaluationSuite {
 
     val actual = plan(inputRow)
     val expectedRow = new GenericRow(Array[Any](expected))
-    if (actual.hashCode() != expectedRow.hashCode()) {
```

Comment:
I think its pretty important to make sure that all of our various internal row implementations have the same hashcode, as this has caused bugs in the past.